### PR TITLE
[bridge] more metrics, logs and instruments 

### PR DESCRIPTION
**Pull request by longbowlu**
## Description 

This PR includes a few things:
1. adds more metrics, logs and instruments to bridge software
2. to do instruments we spin out `fn handle_execution_task` and `fn handle_signing_task` from the while loops.
3. revise some existing metrics such that all errors starts with name `bridge_err`
4. fix a bug where we exit the on chain execution task if the task is already handled. Since it's now in `fn handle_execution_task` this becomes a non-issue
5. added client command `DepositNativeEtherOnEth` which is self-descriptive. I needed this when testing the change locally. The change is small and straightforward so i didn't bother to separate it to its own PR
6. update solidity deploy configs. for 31337.json, change the source chain id to 12. Also add 31338 and make it match the local deploy script's set up.

## Test plan 

Tested things locally including metrics.

---

## Release notes

Check each box that your changes affect. If none of the boxes relate to your changes, release notes aren't required.

For each box you select, include information after the relevant heading that describes the impact of your changes that a user might notice and any actions they must take to implement updates. 

- [ ] Protocol: 
- [ ] Nodes (Validators and Full nodes): 
- [ ] Indexer: 
- [ ] JSON-RPC: 
- [ ] GraphQL: 
- [ ] CLI: 
- [ ] Rust SDK:

### CODE REVIEW

File: bridge/evm/deploy_configs/31337.json [hunk 1]
```diff
@@ -2,9 +2,9 @@
   "committeeMemberStake": [2500, 2500, 2500, 2500],
   "committeeMembers": ["0x68b43fd906c0b8f024a18c56e06744f7c6157c65", "0xacaef39832cb995c4e049437a3e2ec6a7bad1ab5", "0x8061f127910e8ef56f16a2c411220bad25d61444", "0x508f3f1ff45f4ca3d8e86cdcc91445f00acc59fc"],
   "minCommitteeStakeRequired": 10000,
-  "sourceChainId": 0,
+  "sourceChainId": 12,
```
Comment (patrickkuo) [inline bridge/evm/deploy_configs/31337.json:5]: 12 is EthCustom, just want to confirm this is the correct chain id for this config?

File: crates/sui-bridge-cli/src/lib.rs [hunk 2]
```diff
@@ -494,6 +495,15 @@ impl LoadedBridgeCliConfig {
 #[derive(Parser)]
 #[clap(rename_all = "kebab-case")]
 pub enum BridgeClientCommands {
+    #[clap(name = "deposit-native-ether-on-eth")]
```
Comment (dariorussi) [inline crates/sui-bridge-cli/src/lib.rs:498]: @Bridgerz can you please take a look at this?

**Review by dariorussi — COMMENTED** [review]
this is reasonable to me but I'd like for @Bridgerz and @patrickkuo to take a look as well

File: crates/sui-bridge/src/orchestrator.rs [hunk 3]
```diff
@@ -108,12 +120,14 @@ where
             let mut actions = vec![];
             for (sui_event, opt_bridge_event) in events.iter().zip(bridge_events) {
                 if opt_bridge_event.is_none() {
-                    // TODO: we probably should not miss any events, warn for now.
-                    warn!("Sui event not recognized: {:?}", sui_event);
+                    // TODO: we probably should not miss any events, log for now.
+                    metrics.sui_watcher_unrecognized_events.inc();
+                    error!("Sui event not recognized: {:?}", sui_event);
```
[the remark this answers is not in the document]
Comment (longbowlu) [reply crates/sui-bridge/src/orchestrator.rs:125]: i think we do. Theoretically we should capture every event from these contracts/packages

File: crates/sui-bridge-cli/src/lib.rs (hunk 2 above)
Comment (longbowlu) [reply crates/sui-bridge-cli/src/lib.rs:498]: it's an alternative of write to contract in etherscan but also works for local node run by anvil

**Review by Bridgerz — APPROVED** [review]
LGTM

**Pull request merged**